Test/DependencyCollector/FunctionalTests/FuncTest/AspNetCore20HttpTests.cs

```diff
@@ -0,0 +1,177 @@
+﻿namespace FuncTest
+{
+    using System;    
+    using System.Linq;
+    using AI;
+    using FuncTest.Helpers;
+    using FuncTest.Serialization;
+    using Microsoft.VisualStudio.TestTools.UnitTesting;
+    using System.Diagnostics;
+    using System.Globalization;
+
+    /// <summary>
+    /// Tests Dependency Collector Functionality for a WebApplication written in Dotnet Core.
+    /// </summary>
+    [TestClass]
+    public class AspNetCore20HttpTests
+    {
+        internal const string AspxCoreAppFolder = ".\\AspxCore20";
+        internal const string AspxCoreTestAppFolder = "..\\TestApps\\AspxCore20\\";
+        internal static DotNetCoreTestWebApplication AspxCoreTestWebApplication { get; private set; }
+
+        public TestContext TestContext { get; set; }
+
+        [ClassInitialize]
+        public static void MyClassInitialize(TestContext testContext)
```

Comment:
I remember having [this issue](https://blogs.msdn.microsoft.com/ploeh/2007/01/06/classcleanup-may-run-later-than-you-think/) with `ClassInitialize` of the second test class in the same project running before `ClassCleanup` of the first class.
In this project it would mean that we try to deploy App to the same port and it can be busy with the previous test app because `ClassCleanup` was not run.
Have you noticed similar behavior here or it's always run in the order as it should?

Reply:
I noticed that the port is different below, so should be no issue. However, please check that there are no other side effects if `ClassCleanup` is not executed before the other `ClassInitialize`.

Reply:
I am guessing the issue with class initiazlie/cleanup ordering is whats causing occasional test failures in aspnet core sdk repo.. not seen an issue in web sdk.. 
i'll address this before or after merge anyway.